tests/Tests/800__culler/culler.robot

```diff
@@ -116,21 +128,21 @@ Spawn Minimal Image
     [Documentation]    Spawn a minimal image
     Begin Web Test
     Launch JupyterHub Spawner From Dashboard
-    Spawn Notebook With Arguments  image=s2i-minimal-notebook  size=Default
+    Spawn Notebook With Arguments  image=s2i-minimal-notebook  size=Small
 
 Get Notebook Culler Pod Name
     [Documentation]    Finds the current culler pod and returns the name
     ${culler_pod} =  OpenShiftLibrary.Oc Get  kind=Pod
-    ...    label_selector=app=jupyterhub-idle-culler  namespace=redhat-ods-applications
-    ${length} =  Get Length  ${culler_pod}
+    ...    label_selector=component.opendatahub.io/name=kf-notebook-controller  namespace=redhat-ods-applications
+    # ${length} =  Get Length  ${culler_pod}
     # Only 1 culler pod, correct one
-    IF  ${length}==1
-        ${culler_pod_name} =  Set Variable  ${culler_pod[0]}[metadata][name]
-    ELSE
+    #IF  ${length}==1
+    ${culler_pod_name} =  Set Variable  ${culler_pod[0]}[metadata][name]
+    #ELSE
         # There can be more than one during rollout
-        Sleep  10s
-        ${culler_pod_name} =  Get Notebook Culler Pod Name
-    END
+    #    Sleep  10s
```

Comment:
@lugi0 do we need the commented code if not can we remove it?

Reply:
I plan on removing it before merging, still need to do a final cleaning pass

Reply:
Done